### PR TITLE
change 'openlayers' dependency to 'ol'

### PR DESCRIPTION
**Pull request by jdroenner**
change 'openlayers' to 'ol' to avoid pulling all openlayers dependencies like the closure compiler

### CODE REVIEW

File: src/app/layers/layer.model.ts [hunk 1]
```diff
@@ -39,9 +39,9 @@ export class VectorData extends LayerData<Array<ol.Feature>> {
...
                    projection: Projection,
                    extent: [number, number, number, number],
-                   source: (Document | Node | GlobalObject | string),
+                   source: (Document | Node | any | string),
                    opt_options?: olx.format.ReadOptions): VectorData {
-        return new VectorData(time, projection, new ol.format.GeoJSON().readFeatures(source, opt_options), extent);
+        return new VectorData(time, projection, new OlFormatGeoJSON().readFeatures(source, opt_options), extent);
     }
 
     constructor(time: Time, projection: Projection, data: Array<ol.Feature>, extent: [number, number, number, number]) {
```
Comment (ChristianBeilschmidt) [inline src/app/layers/layer.model.ts]: Why does `ol.Feature` work if you changed the above from `ol.format.GeoJson` to `OlFormatGeoJSON`?

File: src/app/layers/symbology/symbology.model.ts [hunk 2]
```diff
@@ -193,11 +199,11 @@ export class SimplePointSymbology extends AbstractVectorSymbology implements ISi
   }
 
   getOlStyle(): ol.style.Style {
-      return new ol.style.Style({
-          image: new ol.style.Circle({
+      return new OlStyleStyle({
```
Comment (ChristianBeilschmidt) [inline src/app/layers/symbology/symbology.model.ts]: Again, is it necessary to use `OlStyleStyle`?

File: src/app/layers/layer.model.ts (hunk 1 above)
Comment (jdroenner) [reply src/app/layers/layer.model.ts]: I guess this is a "feature" of the typings.
The documentation for the module (https://www.npmjs.com/package/ol) states that you have to import everything explicitly.

File: src/app/layers/symbology/symbology.model.ts (hunk 2 above)
Comment (jdroenner) [reply src/app/layers/symbology/symbology.model.ts]: yes

**Comment by jdroenner** [comment]
Please read the documentation of the ol module: https://www.npmjs.com/package/ol

**Comment by ChristianBeilschmidt** [comment]
This is very strange.

**Pull request merged**